.github/workflows/snapshot.yml

```
@@ -0,0 +1,30 @@
+        branches:
+            - main
+        paths:
+            - 'src/**'
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+    release:
+        name: Release
+        runs-on: ubuntu-latest
+        needs: []
```

actionlint (1.7.4)

"needs" section should not be empty

(syntax-check)